File: KCD/HMMasterSlotItemCommand.m
//
//  HMMasterSlotItemCommand.m
//  KCD
//
//  Created by Hori,Masaki on 2014/02/16.
//  Copyright (c) 2014年 Hori,Masaki. All rights reserved.
//

#import "HMMasterSlotItemCommand.h"

@implementation HMMasterSlotItemCommand
- (NSString *)dataKey
{
	return @"api_data.api_mst_slotitem";
}
- (void)execute
{
	[self commitJSONToEntityNamed:@"MasterSlotItem"];
}
@end

File: KCD/HMMasterSlotItemCommand.swift
//
//  HMMasterSlotItemCommand.swift
//  KCD
//
//  Created by Hori,Masaki on 2015/01/11.
//  Copyright (c) 2015年 Hori,Masaki. All rights reserved.
//

import Cocoa

class HMMasterSlotItemCommand: HMJSONCommand {
	override var dataKey: String {
		return "api_data.api_mst_slotitem"
	}
	override func execute() {
		super.commitJSONToEntityNamed("MasterSlotItem")
	}
}
